### ISSUE TITLE
'event does not exist' issue.

### ISSUE THREAD

**Issue by sahilbhat94** [mardurhack/quickhpc#5]
![screenshot from 2018-11-21 00-31-20](https://user-images.githubusercontent.com/38889527/48821139-0c8e6980-ed26-11e8-82f3-a0fd58717277.png)
Hello,
The tool installed on my system. But when I try to run the tool referring to the Readme file, the tool gives the error: "Event does not exist". when I run the command 'papi_avail', it gives me almost 60 events that are available on my system but, when I try to run the tool for an event that is available, it gives me event does not exist error. 

Any help on this ??

Thanks!!

**Comment by shohidulcse1** [comment]
Hi Sahil,
Thanks for your post; I would be happy to know the answer too.
I am wondering how I can run 'papi_avail' command. Should I run it into the command prompt?

Thanks and I appreciate your time!!

**Comment by elchiapp** [comment]
Mmm no idea how to solve this. Things that come to mind: the event is actually not available, even though it shows up in the list, or there's some whitespace issue in the file so that the event's name is incorrectly parsed.

As a test can you run the tool with any other of those events?

**Comment by elchiapp** [comment]
Closing this since there's no response.

**Comment by Jun03** [comment]
Re: I got a similar issue, tried with different events but got the same error: "Event does not exist"
@sahilbhat94 were you able to resolve the issue?
@chpmrc  I've tried to install quickhpc and papi multiple times on different machines, but not able to make it work. is there a need to do something else other than what is mentioned in the build section of the readme, like install some other libraries, or set environment variables, etc? 
NOTE:
I'm not able to use quickhpc with papi 5.6.0(or higher versions)(got a different error during compilation)
![image](https://user-images.githubusercontent.com/22564912/99667565-99de3300-2a92-11eb-882b-c44152cdcce1.png)

When I use PAPI 5.4.1, make runs correctly but when I try to use quickhpc, I get the "Event does not exist" error
![image](https://user-images.githubusercontent.com/22564912/99668058-3dc7de80-2a93-11eb-9013-4695a274d3e6.png)
The highlighted line is my debug statement which I used to print the error code which is returned to papi_add_event(). [link](https://icl.cs.utk.edu/papi/docs/d7/d40/group__ret__codes.html#ga2c179a2d215cc96ec6cec4463d8f679f) specifies error code meanings.
Let me know if I should provide some other info.
